### ISSUE TITLE
Controlled closing

### ISSUE THREAD

**Issue by Darren120** [nandorojo/zeego#57]
Hello, is there a way to control when the menu / dropdown closes?
I want to perform async operation and would like to show a loading indicator and then close the menu

**Comment by nandorojo** [comment]
I don't believe we support this now, but we may be able to. I'm not sure if our Android library supports it though. But iOS and Web do I think
